<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modCalcPlanetsDlg</class>
 <widget class="QWidget" name="modCalcPlanetsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>542</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>9</number>
   </property>
   <item>
    <widget class="KTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="interact">
      <attribute name="title">
       <string>Interactive Mode</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>9</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Input Parameters</string>
         </property>
         <layout class="QHBoxLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>9</number>
          </property>
          <item>
           <layout class="QGridLayout">
            <property name="margin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>6</number>
            </property>
            <item row="1" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Date and time:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="3">
             <widget class="KPushButton" name="LocationButton">
              <property name="text">
               <string>Greenwich, United Kingdom</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="textLabel1">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Solar system body:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>151</width>
                <height>21</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="1">
             <widget class="KComboBox" name="PlanetComboBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <item>
               <property name="text">
                <string>Mercury</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Venus</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Earth</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Mars</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Jupiter</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Saturn</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Uranus</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Neptune</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Pluto</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Moon</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Sun</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="1" colspan="2">
             <widget class="QDateTimeEdit" name="DateTimeBox"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Equatorial Coordinates</string>
           </property>
           <layout class="QGridLayout">
            <property name="margin">
             <number>9</number>
            </property>
            <property name="spacing">
             <number>6</number>
            </property>
            <item row="1" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Dec:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>RA:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="dmsBox" name="RABox">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="dmsBox" name="DecBox">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Horizontal Coordinates</string>
           </property>
           <layout class="QGridLayout">
            <property name="margin">
             <number>9</number>
            </property>
            <property name="spacing">
             <number>6</number>
            </property>
            <item row="1" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Alt:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Az:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="dmsBox" name="AzBox">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="dmsBox" name="AltBox">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Heliocentric Ecliptic Coordinates</string>
           </property>
           <layout class="QGridLayout">
            <property name="margin">
             <number>9</number>
            </property>
            <property name="spacing">
             <number>6</number>
            </property>
            <item row="2" column="2">
             <widget class="QLineEdit" name="HelioDistBox">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Distance (AU):</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Latitude:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Longitude:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" colspan="2">
             <widget class="dmsBox" name="HelioLongBox">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1" colspan="2">
             <widget class="dmsBox" name="HelioLatBox">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string>Geocentric Ecliptic Coordinates</string>
           </property>
           <layout class="QGridLayout">
            <property name="margin">
             <number>9</number>
            </property>
            <property name="spacing">
             <number>6</number>
            </property>
            <item row="2" column="2">
             <widget class="QLineEdit" name="GeoDistBox">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1" colspan="2">
             <widget class="dmsBox" name="GeoLatBox">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Longitude:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Latitude:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Distance (AU):</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" colspan="2">
             <widget class="dmsBox" name="GeoLongBox">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="batch">
      <attribute name="title">
       <string>Batch Mode</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>9</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Select Parameters in Input File</string>
         </property>
         <layout class="QVBoxLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>9</number>
          </property>
          <item>
           <layout class="QHBoxLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QCheckBox" name="PlanetCheckBatch">
              <property name="text">
               <string>Solar system body:</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="KComboBox" name="PlanetComboBoxBatch">
              <item>
               <property name="text">
                <string>Mercury</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Venus</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Earth</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Mars</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Jupiter</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Saturn</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Uranus</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Neptune</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Pluto</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Moon</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Sun</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <layout class="QGridLayout">
              <property name="margin">
               <number>0</number>
              </property>
              <property name="spacing">
               <number>6</number>
              </property>
              <item row="0" column="0">
               <widget class="QCheckBox" name="UTCheckBatch">
                <property name="text">
                 <string>UT:</string>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QTimeEdit" name="UTBoxBatch"/>
              </item>
              <item row="1" column="0">
               <widget class="QCheckBox" name="DateCheckBatch">
                <property name="text">
                 <string>Date:</string>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QDateEdit" name="DateBoxBatch"/>
              </item>
             </layout>
            </item>
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QGridLayout">
              <property name="margin">
               <number>0</number>
              </property>
              <property name="spacing">
               <number>6</number>
              </property>
              <item row="1" column="0">
               <widget class="QCheckBox" name="LatCheckBatch">
                <property name="text">
                 <string>Latitude:</string>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QCheckBox" name="LongCheckBatch">
                <property name="text">
                 <string>Longitude:</string>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="dmsBox" name="LongBoxBatch"/>
              </item>
              <item row="1" column="1">
               <widget class="dmsBox" name="LatBoxBatch"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string>Select Coordinate System for Output File</string>
         </property>
         <layout class="QGridLayout">
          <property name="margin">
           <number>9</number>
          </property>
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="0" column="0">
           <widget class="QCheckBox" name="HelioEclCheckBatch">
            <property name="text">
             <string>Heliocentric ecliptic</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="EquatorialCheckBatch">
            <property name="text">
             <string>Equatorial</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="GeoEclCheckBatch">
            <property name="text">
             <string>Geocentric ecliptic</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="HorizontalCheckBatch">
            <property name="text">
             <string>Horizontal </string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_8">
         <property name="title">
          <string>Other Parameters for Output File</string>
         </property>
         <layout class="QHBoxLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>9</number>
          </property>
          <item>
           <widget class="QRadioButton" name="AllRadioBatch">
            <property name="text">
             <string>All parameters</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="InputRadioBatch">
            <property name="text">
             <string>Only parameters in input file</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_9">
         <property name="title">
          <string>Select Filenames</string>
         </property>
         <layout class="QGridLayout">
          <property name="margin">
           <number>9</number>
          </property>
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Input file:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="KUrlRequester" name="InputFileBoxBatch"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Output file:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="KUrlRequester" name="OutputFileBoxBatch"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>151</width>
             <height>21</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="KPushButton" name="RunButtonBatch">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Run</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>131</width>
             <height>21</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>dmsBox</class>
   <extends>QLineEdit</extends>
   <header>widgets/dmsbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>LocationButton</tabstop>
  <tabstop>PlanetComboBox</tabstop>
  <tabstop>RABox</tabstop>
  <tabstop>DecBox</tabstop>
  <tabstop>AzBox</tabstop>
  <tabstop>AltBox</tabstop>
  <tabstop>HelioLongBox</tabstop>
  <tabstop>HelioLatBox</tabstop>
  <tabstop>HelioDistBox</tabstop>
  <tabstop>GeoLongBox</tabstop>
  <tabstop>GeoLatBox</tabstop>
  <tabstop>GeoDistBox</tabstop>
  <tabstop>PlanetComboBoxBatch</tabstop>
  <tabstop>UTCheckBatch</tabstop>
  <tabstop>UTBoxBatch</tabstop>
  <tabstop>DateCheckBatch</tabstop>
  <tabstop>LatCheckBatch</tabstop>
  <tabstop>LongCheckBatch</tabstop>
  <tabstop>LongBoxBatch</tabstop>
  <tabstop>LatBoxBatch</tabstop>
  <tabstop>HelioEclCheckBatch</tabstop>
  <tabstop>EquatorialCheckBatch</tabstop>
  <tabstop>GeoEclCheckBatch</tabstop>
  <tabstop>HorizontalCheckBatch</tabstop>
  <tabstop>AllRadioBatch</tabstop>
  <tabstop>InputRadioBatch</tabstop>
  <tabstop>OutputFileBoxBatch</tabstop>
  <tabstop>InputFileBoxBatch</tabstop>
  <tabstop>PlanetCheckBatch</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
